specs/grafana-plugin-clickhouse-datasource.spec

```diff
@@ -45,7 +45,7 @@
 Summary:              Clickhouse datasource for Grafana
 Name:                 grafana-plugin-clickhouse-datasource
 Version:              2.1.0
-Release:              0%{?dist}
+Release:              1%{?dist}
```

Comment:
I think we should not bump the package release. It still in the testing repo, so we can just replace the previously built version.

Reply:
This release bump also should be reveted.

Reply:
☝️